test/int/wizards/breakpoints/implementation/breakpointsAssertions.ts

```diff
@@ -0,0 +1,102 @@
+            format: {
+                parameters: true,
+                parameterTypes: true,
+                parameterNames: true,
+                line: true,
+                module: true
+            }
+        });
+
+        this.validateStackTraceResponse(stackTraceResponse);
+
+        const formattedExpectedStackTrace = expectedStackTrace.replace(/^\s+/gm, ''); // Remove the white space we put at the start of the lines to make the stack trace align with the code
+        const actualStackTrace = this.extractStackTrace(stackTraceResponse);
+        assert.equal(actualStackTrace, formattedExpectedStackTrace, `Expected the stack trace when hitting ${breakpoint} to be:\n${formattedExpectedStackTrace}\nyet it is:\n${actualStackTrace}`);
+
+        // const scopesResponse = await this._internal.client.scopesRequest({ frameId: stackTraceResponse.body.stackFrames[0].id });
+        /// const scopes = scopesResponse.body.scopes;
+        await this._internal.client.continueRequest();
+        if (isThisV2) {
+            // TODO: Is getting this event on V2 a bug? See: Continued Event at https://microsoft.github.io/debug-adapter-protocol/specification
+            await this._breakpointsWizard.waitUntilJustResumed();
+        }
+
+        await actionResult;
+    }
+
+    private extractStackTrace(stackTraceResponse: DebugProtocol.StackTraceResponse): string {
+        return stackTraceResponse.body.stackFrames.map(f => this.printStackTraceFrame(f)).join('\n');
+    }
+
+    private printStackTraceFrame(frame: DebugProtocol.StackFrame): string {
+        let frameName = frame.name;
+        if (isThisV1) {
+            // V1 currently has a bug where line numbers are off by 1
+            frameName = frameName.replace(`Line ${frame.line - 1}`, `Line ${frame.line}`);
+        }
+
+        return `${frameName}:${frame.column}${frame.presentationHint && frame.presentationHint !== 'normal' ? ` (${frame.presentationHint})` : ''}`;
+    }
+
+    private validateStackTraceResponse(stackTraceResponse: DebugProtocol.StackTraceResponse) {
+        expect(stackTraceResponse.success, `Expected the response to the stack trace request to be succesful yet it failed: ${JSON.stringify(stackTraceResponse)}`).to.equal(true);
+        expect(stackTraceResponse.body.totalFrames, `The number of stackFrames was different than the value supplied on the totalFrames field`)
+            .to.equal(stackTraceResponse.body.stackFrames.length);
+        stackTraceResponse.body.stackFrames.forEach(frame => {
+            // Warning: We don't currently validate frame.source.path
+            const expectedSourceNameAndLine = ` [${frame.source.name}] Line ${frame.line}`;
+            if (isThisV2) {
+                // V1 has a bug where frame.name is not coherent with frame.line
+                expect(frame.name, 'Expected the formatted name to match the source name and line supplied as individual attributes').to.endsWith(expectedSourceNameAndLine);
```

Comment:
Same, this can be removed once Microsoft/vscode-chrome-debug-core#426 is in place

Reply:
Removed!